waspc/cli/src/Wasp/Cli/Command/CreateNewProject.hs

```diff
@@ -44,81 +51,44 @@ parseProjectInfo name
   where
     appName = kebabToCamelCase name
 
-createNewProject :: String -> Command ()
-createNewProject name =
-  case parseProjectInfo name of
-    Right projectInfo -> createNewProject' projectInfo
-    Left parsedError ->
-      throwError $
-        CommandError "Project creation failed" parsedError
-
-createNewProject' :: ProjectInfo -> Command ()
-createNewProject' (ProjectInfo projectName appName) = do
+createAndPopulateWaspProjectDir :: ProjectInfo -> Command ()
+createAndPopulateWaspProjectDir projectInfo = do
+  absWaspProjectDir <- getAbsoluteWaspProjectDir projectInfo
+  liftIO $ do
+    initializeProjectFromSkeleton absWaspProjectDir
+    writeMainWaspFile absWaspProjectDir projectInfo
+
+printInfoMessages :: ProjectInfo -> IO ()
+printInfoMessages (ProjectInfo projectName _) = do
+  putStrLn $ Term.applyStyles [Term.Green] ("Created new Wasp app in ./" ++ projectName ++ " directory!")
+  putStrLn "To run it, do:"
+  putStrLn ""
+  putStrLn $ Term.applyStyles [Term.Bold] ("    cd " ++ projectName)
+  putStrLn $ Term.applyStyles [Term.Bold] "    wasp start"
+  putStrLn ""
+  putStrLn Command.Common.alphaWarningMessage
+
+getAbsoluteWaspProjectDir :: ProjectInfo -> Command (Path System Abs (Dir WaspProjectDir))
+getAbsoluteWaspProjectDir (ProjectInfo projectName _) = do
   absCwd <- liftIO getCurrentDirectory
-  waspProjectDir <- case SP.parseAbsDir $ absCwd FP.</> projectName of
+  case SP.parseAbsDir $ absCwd FP.</> projectName of
+    Right sp -> return sp
     Left err ->
       throwError $
         CommandError
           "Project creation failed"
-          ( "Failed to parse absolute path to wasp project dir: "
-              ++ show err
-          )
-    Right sp -> return sp
-  liftIO $ do
-    createDirectorySP waspProjectDir
-    writeFileSP (waspProjectDir </> mainWaspFileInWaspProjectDir) mainWaspFileContent
-    writeFileSP (waspProjectDir </> gitignoreFileInWaspProjectDir) gitignoreFileContent
-    writeFileSP
-      (waspProjectDir </> Common.dotWaspRootFileInWaspProjectDir)
-      "File marking the root of Wasp project."
-
-  let extCodeDir = waspProjectDir </> Common.extCodeDirInWaspProjectDir
-  liftIO $ do
-    createDirectorySP extCodeDir
-    dataDir <- Wasp.Data.getAbsDataDirPath
-
-    let copyTemplateFile' = copyTemplateFile dataDir extCodeDir
+          ("Failed to parse absolute path to wasp project dir: " ++ show err)
 
-    writeFileSP (extCodeDir </> waspignoreFileInExtCodeDir) waspignoreFileContent
+initializeProjectFromSkeleton :: Path' Abs (Dir Common.WaspProjectDir) -> IO ()
```

Comment:
I took the term from linux systems (i.e., the `/etc/skel` folder), but yeah, it is weird if you've never seen it.

I'd probabaly rather add a comment explaining it than rename it to `initializeProjectFromTemplates` because we use the word "template" for something else throughout the codebase.